src/frontends/tensorflow_common/src/op/multinomial.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_op_table.hpp"
+#include "openvino/opsets/opset13.hpp"
```

Comment:
please use explicit include of Multinomial operation. We avoid common include usage now.

Reply:
Done